model/src/main/java/de/danoeh/antennapod/model/feed/Feed.java

```diff
@@ -176,10 +176,11 @@ public Feed(String url, String lastModified, String title) {
     /**
      * This constructor is used for requesting a feed download (it must not be used for anything else!). It should be
      * used if the title of the feed is already known.
+     * Todo: check if the change to DEFAULT is correct
```

Comment:
I would say the change is correct:

> for requesting a feed download (it must not be used for anything else!)

So it shouldn't really matter, but I think default is always a good default :)